Mailbox.Tests/DataLoaderTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Mailbox.Tests
+{
+    [TestClass]
+    public class DataLoaderTests
+    {
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void Load_StreamIsNull_ThrowNullException()
+        {
+            DataLoader dL = new DataLoader(null);
+        }
+
+        [TestMethod]
+        public void Load_NotValidJson_ReturnsNull()
+        {
+            MemoryStream stream = GenerateTextFile();
+            DataLoader dL = new DataLoader(stream);
```

Comment:
DataLoader is disposable and should be cleaned up.
```suggestion
            using DataLoader dL = new DataLoader(stream);
```